core/validation/initial_check.py

```diff
@@ -0,0 +1,168 @@
+
+    sheet_a1 = workbook.get("1 - Start Here")
+    if sheet_a1 is not None:
+        details_dict["Form Version"] = sheet_a1.iloc[6][1]
+        details_dict["Reporting Period"] = sheet_a1.iloc[4][1]
+
+    sheet_a2 = workbook.get("2 - Project Admin")
+    if sheet_a2 is not None:
+        details_dict["Fund Type"] = sheet_a2.iloc[5][4]
+        details_dict["Place Name"] = sheet_a2.iloc[6][4]
+
+    # specific to Round 3
+    details_dict["Expected Form Version"] = "Town Deals and Future High Streets Fund Reporting Template (v3.0)"
+    details_dict["Reporting Round"] = 3
+
+    return details_dict
+
+
+def pre_transformation_check(submission_details: dict[str, str]) -> list[vf.ValidationFailure]:
+    """
+    Perform pre-transformation checks on the given workbook.
+
+    :param submission_details: A dictionary containing submission details.
+    :return: A list of validation failures encountered during pre-transformation checks.
+    """
+    pre_transformation_failures = []
+
+    if "Missing Sheets" in submission_details:
+        return [vf.EmptySheetFailure(empty_sheet=empty_sheet) for empty_sheet in submission_details["Missing Sheets"]]
+
+    check_form_version(submission_details, pre_transformation_failures)
+    check_reporting_round(submission_details, pre_transformation_failures)
+    check_place_name_input(
+        submission_details,
+        pre_transformation_failures,
+    )
+    check_fund_type_input(submission_details, pre_transformation_failures)
+
+    return pre_transformation_failures
+
+
+def check_form_version(submission_details: dict[str, str], failures: list[vf.ValidationFailure]) -> None:
+    """
+    Check if the form type is correct.
+
+    :param submission_details: A dictionary containing submission details.
+    :param failures: A list to store validation failures.
+    :return: None
+    """
+
+    actual_form_version = submission_details["Form Version"]
+
+    if actual_form_version != submission_details["Expected Form Version"]:
+        failures.append(
+            vf.WrongFormVersionFailure(
+                expected_form_version=submission_details["Expected Form Version"],
+                actual_form_version=actual_form_version,
+            )
+        )
+
+
+def check_reporting_round(submission_details: dict[str, str], failures: list[vf.ValidationFailure]) -> None:
+    """
+    Check if the round value is correct.
+
+    :param submission_details: A dictionary containing submission details.
+    :param failures: A list to store validation failures.
+    :return: None
+    """
+    reporting_period = submission_details["Reporting Period"]
+
+    if reporting_period not in REPORTING_PERIOD_DICT:
+        failures.append(
+            vf.InvalidReportingPeriodFailure(
+                invalid_reporting_period=reporting_period,
+                valid_reporting_periods=REPORTING_PERIOD_DICT.keys(),
+            )
+        )
+
+        return
+
+    round_value = REPORTING_PERIOD_DICT[reporting_period]
```

Comment:
I think it'd be more useful to actually return the date period strings here rather than the reporting round number (since that's our concept, users may only understand in terms of the date range they put on the spreadsheet form). So could just not lookup the values from REPORTING_PERIOD_DICT for this bit, just effectively use the `reporting_period`